iChef-WebApi/src/test/java/com/kitchen/iChef/Service/UserServiceTest.java

```diff
@@ -0,0 +1,271 @@
+package com.kitchen.iChef.Service;
+import com.kitchen.iChef.Domain.AppUser;
+import com.kitchen.iChef.Domain.Token;
+import com.kitchen.iChef.Repository.UserRepository;
+import com.kitchen.iChef.Service.Hashing.BCryptPasswordEncoder;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@ExtendWith(MockitoExtension.class)
+class UserServiceTest {
+
+    @InjectMocks
+    UserService userService;
+    @Mock
+    UserRepository userRepository;
+    @Mock
+    TokenService tokenService;
+    @Test
+    @DisplayName("Test Add Success Case")
+    @Order(1)
+    void add_success() {
+        AppUser appUser = new AppUser();
```

Comment:
Can we extract this in a separate method? We use this in every test and it makes tests longer than needed